BlockchainTests/GeneralStateTests/stSubroutine/simpleSubroutine.json

```diff
@@ -2,11 +2,11 @@
     "simpleSubroutine_d0g0v0_Berlin" : {
         "_info" : {
             "comment" : "",
-            "filling-rpc-server" : "evm version 1.9.23-unstable-0efc69a9-20201022",
```

Comment:
why this file is not removed ?

Reply:
That file is included so that a client that forgets to turn off 2315 will fail.

Reply:
(Theoretically i think that will be caught by `opcodeTests` though, correct? I included this because my understanding of what @holiman was looking for was a specific test that would fail if a client implemented 2315)

Reply:
I don't want to have a whole directory just for a single test, it makes things messy. I'll move the test to stBadOpcode.

Reply:
Makes sense!

Reply:
Updated in fdd4da0.